expression/builtin_string.go

```diff
@@ -2050,39 +2050,48 @@ type eltFunctionClass struct {
 }
 
 func (c *eltFunctionClass) getFunction(args []Expression, ctx context.Context) (builtinFunc, error) {
-	sig := &builtinEltSig{newBaseBuiltinFunc(args, ctx)}
-	return sig.setSelf(sig), errors.Trace(c.verifyArgs(args))
+	if argsErr := c.verifyArgs(args); argsErr != nil {
+		return nil, errors.Trace(argsErr)
+	}
+	argTps := make([]evalTp, 0, len(args))
+	argTps = append(argTps, tpInt)
+	for i := 1; i < len(args); i++ {
+		argTps = append(argTps, tpString)
+	}
+	bf, err := newBaseBuiltinFuncWithTp(args, ctx, tpString, argTps...)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	for _, arg := range args[1:] {
+		argType := arg.GetType()
+		//setBinFlagOrBinStr(argType, bf.tp)
```

Comment:
@hanfei1991 I am not sure whether 'Charset', 'Collate' and 'Flag' changes according to parameters in builtin 'ELT'. I am trying to feature out how it works in Mysql. Do you have any ideas about this ?